### DISCUSSION TITLE
Welcome to RcppSMC Discussions!

### DISCUSSION THREAD

**Discussion by eddelbuettel** [rcppsmc/rcppsmc#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LeahPrice** [comment]
Hi! I'm a lecturer at Queensland University of Technology and I'm interested in Bayesian computational statistics methods, including sequential Monte Carlo. I started my RcppSMC journey as a student in Google Summer of Code 2017 and I'm happy to be working with the community on ongoing developments.

**Reply by ilyaZar** [reply]
Hi! I am a Ph.D. student at the Statistics and Econometrics department at the University of Cologne (Germany). I am interested in Bayesian statistical modeling techniques for multivariate panel data and Monte Carlo methods (PMCMC, SMC). Currently, I participate in this years Google Summer of Code and hopefully can contribute to the great work all of you have done so far!

**Comment by adamjohansen** [comment]
Hello. I'm a professor in the Statistics Department at The University of Warwick with wide-ranging interests in computational statistical methodology and, particularly, sequential Monte Carlo. I've been involved with RcppSMC since the beginning and look forward to future developments.

**Reply by eddelbuettel** [reply]
Howdy. I am a software engieer at TileDB, and an adjunct Clinical Professor in Statistics at Illinois, and long-time R and Debian contributor. I once started RcppSMC because Adam's JSS paper on SMC looked like a really good fit, Adam jumped in pretty mich rightaway, and we (for a few years now including Leah) have been trying to make sense of it ever since :)    

I also turned on this discussion panel with its default robot-generated first issue so here we all are :joy:
